plugins/base/src/main/java/edu/wpi/first/shuffleboard/plugin/base/widget/GraphWidget.java

```diff
@@ -1,5 +1,12 @@
 package edu.wpi.first.shuffleboard.plugin.base.widget;
 
+import com.google.common.collect.ImmutableList;
```

Comment:
Import order should be

`edu.wpi.first.shuffleboard.*`
`edu.wpi.*`
[misc imports eg `com.*`, `de.*`, `edu.*`, grouped and alphabetical]
`java.*`
`javafx.*`